xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -21,6 +21,9 @@ limitations under the License.
 #include <gtest/gtest.h>
 #include "absl/functional/any_invocable.h"
 #include "absl/strings/str_replace.h"
+#if TENSORFLOW_USE_ROCM
```

Comment:
I think this is not needed given the include on line 45 (linter complains about this internally)

Reply:
You're right. I just removed it.